docs/site/downloads/cldr-45.md

```diff
@@ -10,7 +10,7 @@ title: CLDR 45 Release Note
 
 ## Overview
 
-Unicode CLDR provides key building blocks for software supporting the world's languages. CLDR data is used by all [major software systems](https://www.google.com/url?q=http://cldr.unicode.org/index%23TOC-Who-uses-CLDR-&sa=D&source=editors&ust=1713652058779208&usg=AOvVaw3r8kLAP0U8srdRrX-YUWlu) (including all mobile phones) for their software internationalization and localization, adapting software to the conventions of different languages.
+Unicode CLDR provides key building blocks for software supporting the world's languages. CLDR data is used by all [major software systems](/index#who-uses-cldr) (including all mobile phones) for their software internationalization and localization, adapting software to the conventions of different languages.
```

Comment:
Some links, like this one, were marked "Change to resolved file (after "&q=", before "&amp;", unescaped)" in the spreadsheet. However, a better solution sometimes, like here, is to use a relative link

Reply:
Definitely!